packages/styles/src/components/accordion.scss

```diff
@@ -1,94 +1,124 @@
+@use './../mixins/icons' as icon-mixins;
+@use './../mixins/type' as type-mixins;
+@use './../mixins/utilities' as utility-mixins;
 
-@use './../mixins/type';
-@use './../mixins/utilities';
-@use './../variables/spacing';
 @use './../variables/components/accordion';
-@use './../variables/components/button';
 
 .accordion-item {
-  border-right: 0;
-  border-left: 0;
-  background-color: transparent;
+  --post-accordion-border-width: #{accordion.$accordion-border-width};
+  --post-accordion-border-color: #{accordion.$accordion-border-color};
+  --post-accordion-header-font-weight: #{accordion.$accordion-header-font-weight};
+  --post-accordion-header-line-height: #{accordion.$accordion-header-line-height};
+  --post-accordion-button-padding: #{accordion.$accordion-button-padding};
+  --post-accordion-icon-size: #{accordion.$accordion-icon-size};
+  --post-accordion-button-cursor: #{accordion.$accordion-button-cursor};
+  --post-accordion-button-disabled-opacity: #{accordion.$accordion-button-disabled-opacity};
+  --post-accordion-button-focus-box-shadow: #{accordion.$accordion-button-focus-box-shadow};
+  --post-accordion-icon-transition: #{accordion.$accordion-icon-transition};
+  --post-accordion-icon-transform: #{accordion.$accordion-icon-transform};
+  --post-accordion-body-padding: #{accordion.$accordion-body-padding};
+  --post-accordion-heading-margin: #{accordion.$accordion-heading-margin};
+  --post-accordion-heading-font-weight: #{accordion.$accordion-heading-font-weight};
+
+  border-block-start: var(--post-accordion-border-width) solid var(--post-accordion-border-color);
+}
 
-  &:last-of-type {
-    border-bottom: 0;
-  }
+.accordion-header {
+  @include type-mixins.font-curve(accordion.$accordion-header-font-curve);
+  font-weight: var(--post-accordion-header-font-weight);
+  line-height: var(--post-accordion-header-line-height);
+  margin: 0;
 }
 
 .accordion-button {
-  @include type.font-curve(accordion.$accordion-font-curve);
+  @include button-mixins.reset-button;
 
-  background-color: transparent;
-  font-weight: accordion.$accordion-button-font-weight;
-  line-height: accordion.$accordion-button-line-height;
+  width: 100%;
+  position: relative;
+  padding-block: var(--post-accordion-button-padding);
+  padding-inline-start: var(--post-accordion-button-padding);
+  padding-inline-end: calc(var(--post-accordion-button-padding) + var(--post-accordion-icon-size));
+  text-align: start;
 
-  &:hover,
-  &:focus {
-    &:not(:disabled) {
-      color: accordion.$accordion-button-hover-color;
-    }
+  &:not(:disabled) {
+    cursor: var(--post-accordion-button-cursor);
   }
 
   &:disabled {
-    opacity: button.$btn-disabled-opacity !important;
+    opacity: var(--post-accordion-button-disabled-opacity);
+  }
+
+  &:focus {
+    outline: none;
+    box-shadow: var(--post-accordion-button-focus-box-shadow);
+  }
+
+  &::after {
+    @include icon-mixins.pi(accordion.$accordion-icon-name);
```

Comment:
It would be nice if you could use the new icon system :)

Reply:
Is there a way to use the new icon system in a Sass file?
The new system works with HTML markups, doesn't it?

Reply:
I opened [this issue](https://github.com/swisspost/design-system/issues/1361) to find a way to use the new icon system within the styles package.